packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/components/PagespeedInsights.astro

```diff
@@ -64,49 +64,55 @@ const FetchURL = Astro.site||"";
 import type { SlButton, SlProgressRing } from "astrolace:types";
 import { generateLighthouseFetchUrl, setProgressRing } from "./webVitalsUtils";
 
+// get the URL to test from the hidden div
 const urlToTest: string = document.getElementById('lighthouse-fetch-url')?.textContent || '';
-
 const lighthouseFetchUrl: string = generateLighthouseFetchUrl(urlToTest);
 
+// get the `Run Test` button and add an event listener to it
 const runTestButton = document.getElementsByName('run-test')[0] as SlButton;
 
-runTestButton.addEventListener('click', async (e) => {
-    e.preventDefault();
+runTestButton.addEventListener('click', async (
+    // I changed the name of the event from `e` to `buttonClickEvent` for @jdtjenkins(Jumper) since "He wants things to be more descriptive and 'Easier to read'".
+    buttonClickEvent
+) => {
+    buttonClickEvent.preventDefault();
```

Comment:
Unless we're _using_ this event for anything, `e` is fine in this case because literally everyone just uses `e` for DOM event listeners 😂 The issue comes with naming custom variables

Reply:
FWIW I flipflop between `e` and `event` for these listeners 👀

Reply:
nope.  its this now.  you asked for it <3